### DISCUSSION TITLE
Name of a stock Plugin

### DISCUSSION THREAD

**Discussion by curtistornade** [demberto/PyFLP#141] [Q&A] [answered]
Hi,
Again thank you a lot for your work.
Just a little question, is it possible to have the name of a `pyflp.plugin._PluginBase` object ?

**Comment by demberto** [comment, marked as answer]
Both `pyflp.channel.Channel` and `pyflp.mixer.Slot` have an `internal_name` attribute. For stock plugins it contains the factory name of the plugin.

*yes, i need to fix the docstring for `pyflp.channel.Channel.internal_name`.*

**Reply by curtistornade** [reply]
Thank you very much 👌
